src/lib/errors.ts

```diff
@@ -23,7 +23,7 @@ export class StatefulOrderError extends Error {
 type RouteErrorCodes = 5;
 
 const routeErrorCodesToMessageOverride = {
-  5: 'This route is not yet supported, please check back soon as we are adding new routes.',
+  5: 'This route will be supported soon. In the meantime please try another asset/chain!',
```

Comment:
i know this isn't localized, but tbf none of the other error messages are and this message is temporary (so lilt may not even have it translated by the time it goes away)